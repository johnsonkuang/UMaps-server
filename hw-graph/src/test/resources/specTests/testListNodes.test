# Test Listing Nodes
# Tests whether the graph can list their nodes

# Create graph
CreateGraph graph1

# Add nodes
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D
AddNode graph1 E

# List the nodes
ListNodes graph1