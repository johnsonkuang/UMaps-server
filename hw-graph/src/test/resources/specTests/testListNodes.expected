# Test Listing Nodes
# Tests whether the graph can list their nodes

# Create graph
created graph graph1

# Add nodes
added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1
added node E to graph1

# List the nodes
graph1 contains: A B C D E